### PR TITLE
Task#12059: init api

### PR DESCRIPTION
**Pull request by nguyenduccanh2501**
### [Redmine link]
https://edu-redmine.sun-asterisk.vn/issues/12059

### CODE REVIEW

**Comment by nguyenduccanh2501** [comment]
ready

File: app/src/main/java/com/sun/music_64/data/model/Genre.java [hunk 1]
```diff
@@ -1,4 +1,37 @@
 package com.sun.music_64.data.model;
 
 public class Genre {
+    private String mName;
+    private String mKey;
+    private int mImageID;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/sun/music_64/data/model/Genre.java]:
```suggestion
    private int mImageId;
```

File: app/src/main/java/com/sun/music_64/data/model/Track.java [hunk 2]
```diff
@@ -1,4 +1,108 @@
 package com.sun.music_64.data.model;
 
-public class Track {
+import java.io.Serializable;
+
+public class Track implements Serializable {
+    private int mID;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/sun/music_64/data/model/Track.java]:
```suggestion
    private int mId;
```

File: app/src/main/java/com/sun/music_64/data/model/Track.java [hunk 3]
```diff
@@ -1,4 +1,108 @@
...
+    private String mGenre;
+    private String mTitle;
+    private long mDuration;
+    private String mArtist;
+    private String mStreamUrl;
+    private String mDownloadUrl;
+    private String mArtworkUrl;
+    private String mDescription;
+    private boolean isDownload;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/sun/music_64/data/model/Track.java]:
```suggestion
    private boolean mIsDownloadable;
```

File: app/src/main/java/com/sun/music_64/data/model/Track.java [hunk 4]
```diff
@@ -1,4 +1,108 @@
...
+    private boolean isFavourite;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/sun/music_64/data/model/Track.java]:
```suggestion
    private boolean mIsFavourite;
```

File: app/src/main/java/com/sun/music_64/screen/SplashActivity.java [hunk 5]
```diff
@@ -11,7 +11,6 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_splash);
         startActivity(new Intent(this, MainActivity.class));
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/sun/music_64/screen/SplashActivity.java]: wrong convention 2.11

**Comment by nguyenduccanh2501** [comment]
ready

File: app/src/main/java/com/sun/music_64/data/model/Track.java [hunk 6]
```diff
@@ -1,4 +1,108 @@
...
+    private int mId;
+    private String mGenre;
+    private String mTitle;
+    private long mDuration;
+    private String mArtist;
+    private String mStreamUrl;
+    private String mDownloadUrl;
+    private String mArtworkUrl;
+    private String mDescription;
+    private boolean mIsDownload;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/sun/music_64/data/model/Track.java]:
```suggestion
    private boolean mIsDownloadable;
```

File: app/src/main/java/com/sun/music_64/screen/SplashActivity.java [hunk 7]
```diff
@@ -11,8 +11,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_splash);
-        startActivity(new Intent(this, MainActivity.class));
+        startActivity(new Intent(SplashActivity.this, MainActivity.class));
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/sun/music_64/screen/SplashActivity.java]: chưa đúng convention 2.11 em

**Pull request merged**